src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -6228,5 +6228,281 @@ void Test()
 }";
             await TestMissingAsync(code);
         }
+
+        [WorkItem(40209, "https://github.com/dotnet/roslyn/issues/40209")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        public async Task TestIntroduceFromNonStaticLocalFunction_AllOccurences_Method()
+        {
+            await TestInRegularAndScriptAsync(
+@"class Program
+{
+    void M(int i)
+    {
+        var x = i.ToString();
+        Local();
+
+        void Local()
```

Comment:
It's less common, but you probably want to test with a nested local function (local function inside another local function).

Reply:
Thank you for the suggestion! Per your comments, I added some nested local function tests in the PR here: #40441